Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/.gitignore

```diff
@@ -0,0 +1,2 @@
+# files generated during the lubuild process
+generated/

```

Comment:
Copy the gitignore from EchoSkillBotComposer https://github.com/microsoft/BotFramework-FunctionalTests/blob/main/Bots/DotNet/Skills/Composer/EchoSkillBotComposer/.gitignore here.

The composer authoring tool generates some files on the first run, we could do this on the build servers but it will complicate the deployments so for now, let's just assume the developer will run composer locally at least once and have the generated files pushed as part of the source.

Reply:
Done. Thanks!

Reply:
Done, thanks!